packages/amplify-cli/src/utils/headless-input-utils.ts

```diff
@@ -29,4 +29,4 @@ export const readHeadlessPayload = async (): Promise<string> => {
   });
 };
 
-export const isYesFlagSet = (context: Context): boolean => context?.exeInfo?.inputParams?.yes;
+export const isYesFlagSet = (context: Context): boolean => context?.exeInfo?.inputParams?.yes || context?.exeInfo?.inputParams?.force;
```

Comment:
Do we need to set this or the change below is sufficient? Tehnically force flag is not the same as yes right?

Reply:
Yeah you are right `--force` is only used for delete